tests/conftest.py

```diff
@@ -87,6 +87,46 @@
 TABLES_TO_KEEP = [x.table_name() for x in data._classes]
 
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "--fast",
+        action='store_true',
+        help="Don't run all test, but a representative set",
+    )
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_generate_tests(metafunc: "pytest.Metafunc") -> None:
+    """
+    For each test marked as parametrize_any run it either as
+    1. if not in fast mode: as if annotated with @parametrize
+    2. if in fast mode: with only one parameter combination, chosen randomly
+    """
+
+    is_fast = metafunc.config.getoption("fast")
+    for marker in metafunc.definition.iter_markers(name="parametrize_any"):
+        variations = len(marker.args[1])
+        if not is_fast or variations<=2:
```

Comment:
Why `<=2`?

Reply:
if there are 0 or 1 variations, we don't touch it?

Reply:
But now it's implemented as _"if there are 0, 1 or 2 variations, we don't touch it"_. Right?

Reply:
```suggestion
        if not is_fast or variations<2:
```

Reply:
indeed, this is wrong